### PR TITLE
fix: remove some toRaw

### PR DESCRIPTION
**Pull request by amritk**
it was bugging me, this makes more sense

### CODE REVIEW

**Comment by marclave** [comment]
@amritk wait why does this work 🤯 what changed 🤯

**Comment by amritk** [comment]
Removed a couple toRaws, but Hans pointed out we can do an even better job

**Comment by amritk** [comment]
Alright now its much better

File: packages/api-client/src/store/workspace.ts [hunk 1]
```diff
@@ -545,7 +545,8 @@ const deleteServer = (serverUid: string, collectionUid: string) => {
 // ---------------------------------------------------------------------------
 
 /** Helper function to import a OpenAPI spec file into the local workspace */
-async function importSpecFile(spec: string | AnyObject) {
+async function importSpecFile(_spec: string | AnyObject) {
```
Comment (marclave) [inline packages/api-client/src/store/workspace.ts:548]: bit of a nit but doesnt underscore usually mean unused for function params?

**Review by marclave — APPROVED** [review]
ah nice now i see why this works :)

**Pull request merged**